test/tests.c

```diff
@@ -126,8 +128,11 @@ double test_true_peak(const char* filename) {
     ebur128_add_frames_double(st, buffer, (size_t) nr_frames_read);
   }
 
-  ebur128_loudness_range(st, &true_peak);
-
+  for (int i = 0; i < file_info.channels; i++) {
```

Comment:
'for' loop initial declarations are only allowed in c99 mode. Consider declaring "i" at the top of the function as is done in the rest of the code.